inscriptions/ISic003682.xml

```diff
@@ -183,11 +180,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM"> <!-- see JP's comment below to check if text div is correct -->
```

Comment:
Two different hands, according to Orsi, so doing it as two text parts like this seems reasonable to me. I cannot find anyindication that this material ever entered the museum; the Taccuini for this year / excavation will ultimately be crucial I think... The other material from this tombwill need adding in due course

Reply:
We could define the different hands in handNote and indicate the handShift in the text, if we want to keet the text running uninterrupted by divs.

Reply:
I would be happy with handNote and handShift (will be applicable to the Taormina stratagoi monster, but I skipped that for now as I lack the detailed data); but we'll still need `<cb>` if nothing else since the two texts are side-by-side

Reply:
Some troubles with the handShift/handNote: am I wrong or is it not allowed in our TEI model to insert a handNotes element?

Reply:
So, up in the handDesc you add a new handNote with an `@xml:id` with some value that makes sense to you, e.g. 'h2'. Then down in the text, at the point at which the hand changes, you put the empty element `<handShift new="#h2"/>`.

Reply:
Ok, but why the validation failed? it seems because of the handNotes (pl.) element...